pkg/common/utils/gang_utils.go

```diff
@@ -57,11 +58,24 @@ func FindAppTaskGroup(appTaskGroups []*interfaces.TaskGroup, groupName string) (
 // GeneratePlaceholderName creates the placeholder name for a pod, pod name can not be longer than 63 chars,
 // taskGroup name and appID will be truncated if they go over 20/28 chars respectively,
 // each taskGroup is assigned with an incremental index starting from 0.
-func GeneratePlaceholderName(taskGroupName, appID string, index int32) string {
-	// taskGroup name no longer than 20 chars
+func GeneratePlaceholderName(taskGroupName, appID string) string {
```

Comment:
NIT: Update the comment to align with the new generation logic.

Reply:
Done